### DISCUSSION TITLE
https://imagekit.io/docs/image-optimization

### DISCUSSION THREAD

**Discussion by giscus** [imagekitio/discussions#57] [Docs]
# https://imagekit.io/docs/image-optimization

Learn how to optimize images for web delivery using ImageKit.io.

https://imagekit.io/docs/image-optimization

**Comment by navjotsharma5500** [comment]
Could you please help me with this? The attachments have been uploaded successfully, but the uploaded pictures are not showing any preview and appear to be corrupted. However, when I download them, the images are perfectly fine. Only on ImageKit the preview is not visible.
Kindly help me with this.
